From b7d4e96379c2a56b9222d4640618576a03e4dafd Mon Sep 17 00:00:00 2001
From: David Kalnischkies <david@kalnischkies.de>
Date: Sun, 29 Jan 2023 13:42:35 +0100
Subject: [PATCH] Include cstdint for uint16_t in our mmap wrapping

In gcc-13 internal includes were reduced exposing our laziness.

Reported-By: gcc-13
Gbp-Dch: Ignore
---
 apt-pkg/contrib/mmap.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apt-pkg/contrib/mmap.cc b/apt-pkg/contrib/mmap.cc
index 642e20473..0568e1cd0 100644
--- a/apt-pkg/contrib/mmap.cc
+++ b/apt-pkg/contrib/mmap.cc
@@ -23,6 +23,7 @@
 #include <apt-pkg/macros.h>
 #include <apt-pkg/mmap.h>
 
+#include <cstdint>
 #include <cstring>
 #include <string>
 #include <errno.h>
-- 
2.40.0

